src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1438,11 +1448,13 @@ public void TestFloor()
             {
                 Helper.AssertTemplateResult("1", "{{ 1.2 | floor }}");
                 Helper.AssertTemplateResult("2", "{{ 2.0 | floor }}");
-                Helper.AssertTemplateResult("183", "{{ 183.357 | floor }}");
+                Helper.AssertTemplateResult("183", "{{ 183.57 | floor }}");

```

Comment:
why change the test input?

Reply:
But we already have 3.5 floor on the very next test line to distinguish from round.